B/BOLT/build_tarballs.jl

```diff
@@ -0,0 +1,169 @@
+using BinaryBuilder, Pkg, LibGit2
```

Comment:
LibGit2 is unused?
```suggestion
using BinaryBuilder, Pkg
```
Probably Pkg too right now, but this one is more likely to be needed from time to time